ouroboros-consensus/ouroboros-consensus-mock/src/Ouroboros/Consensus/Mock/Ledger/Block.hs

```diff
@@ -112,8 +112,19 @@ data SimpleBlock' c ext ext' = SimpleBlock {
       simpleHeader :: Header (SimpleBlock' c ext ext')
     , simpleBody   :: SimpleBody
     }
-  deriving stock    (Generic, Show, Eq)
-  deriving anyclass (Serialise)
+  deriving (Generic, Show, Eq)
+
+instance (SimpleCrypto c, Serialise ext') => Serialise (SimpleBlock' c ext ext') where
+  encode (SimpleBlock hdr body) = mconcat [
+        CBOR.encodeListLen 2
+      , encode hdr
+      , encode body
+      ]
+  decode = do
+      CBOR.decodeListLenOf 2
+      hdr  <- decode
+      body <- decode
+      return (SimpleBlock hdr body)
```

Comment:
I went for this instead of the generic instance so it's easier to estimate the size of the block.